CONTRIBUTING.md

```diff
@@ -72,38 +72,23 @@ setup.py  # Project is pip-installable, project requirements managed here.
    cd weather-tools
    pip install -e ."[dev]"
    ```
-   > Note: If the execution of `pip install -e ."[dev]"` command takes much time, try upgrading your pip version:
-   > `pip install --upgrade pip`.
+    > Note: If the execution of `pip install -e ."[dev]"` command takes much time, try upgrading your pip version.
 
-3. Install `gcloud`, the Google Cloud CLI, following these [instructions](https://cloud.google.com/sdk/docs/install).
+1. Install `gcloud`, the Google Cloud CLI, following these [instructions](https://cloud.google.com/sdk/docs/install).
 
-4. Acquire adequate permissions for the project.
+1. Acquire adequate permissions for the project.
     * Run `gcloud auth application-default login`.
 
     * Make sure your account has *write* permissions to the storage bucket as well as the permissions to create a
       Dataflow job.
 
     * Make sure that both Dataflow and Cloud Storage are enabled on your Google Cloud Platform project.
 
-5. Install ecCodes and MetView on to your local environment (Anaconda seems to be the only supported way).
-   ```shell
-   conda install eccodes metview-batch -c conda-forge
-   ```
-
-   If you're developing on a Mac, you can also use Homebrew:
-   ```shell
-   brew install eccodes
-   brew install metview 
-   ```
-
-   Linux users: The debian package for eccodes works, but metview is
-   broken ([metview-python#19](https://github.com/ecmwf/metview-python/issues/19)).
-
 ### Windows Developer Instructions
 
 Windows support for each CLI is currently under development (
 See [#64](https://github.com/google/weather-tools/issues/64)). However, there are workarounds available for running the
-nweather tools outside of installation with `pip`.
+weather tools outside of installation with `pip`.
```

Comment:
nice

Reply:
Performed `git rebase` and resolved conflict errors.